src/main/steps/applicant1/confirm-your-joint-application/content.ts

```diff
@@ -1,158 +1,180 @@
+  line5: '<strong> Applicant 1 </strong>',
+  line6: `${userCase?.applicant1FirstNames} ${userCase?.applicant1MiddleNames} ${userCase?.applicant1LastNames}`,
+  line7: '<strong> Applicant 2 </strong>',
+  line8: `${userCase?.applicant2FirstNames} ${userCase?.applicant2MiddleNames} ${userCase?.applicant2LastNames}`,
+  subHeading2: `About the ${isDivorce ? 'marriage' : 'civil partnership'}`,
+  line10: `These details are copied directly from the ${isDivorce ? 'marriage' : 'civil partnership'} certificate,
+     or the translation of the certificate, if it’s not in English. The names on the certificate are the names the
+      applicant and respondent used before the ${isDivorce ? 'marriage' : 'civil partnership'}.`,
+  line11: `<strong>Who the ${isDivorce ? 'marriage' : 'civil partnership'} is between</strong>`,
+  line12: `${userCase?.applicant1FullNameOnCertificate}  and ${userCase?.applicant2FullNameOnCertificate}
+      (as shown on the ${isDivorce ? 'marriage' : 'civil partnership'} certificate)`,
+  line13: `<strong> Where the ${isDivorce ? 'marriage' : 'civil partnership'} took place</strong>`,
+  line14: `${userCase?.ceremonyPlace}`,
+  line15: `<strong>Date of ${isDivorce ? 'marriage' : 'civil partnership'}</strong>`,
+  line16: `${getFormattedDate(userCase?.relationshipDate)}`,
+  subHeading3: 'Why the court can deal with the case (jurisdiction)',
+  line17: 'The courts of England and Wales have the legal power (jurisdiction) to deal with this case because:',
+  connectionBulletPoints: userCase ? connectionBulletPointsTextForRespondent(userCase.connections!) : [],
+  jurisdictionsMoreDetails:
+    `The courts of England or Wales must have the jurisdiction (the legal power) to be able to ${
+      isDivorce ? 'grant a divorce' : 'end a civil partnership'
+    }.
+    The applicants confirmed that the legal statement(s) in the application apply to either or both the applicants. Each legal statement includes some or all of the following legal connections to England or Wales.` +
+    '<br><br>' +
+    jurisdictionMoreDetailsContent(userCase?.connections, isDivorce).connectedToEnglandWales,
+  whatThisMeans: 'What this means',
+  subHeading4: 'Other court cases',
+  line18: `The court needs to know about any other court cases relating to the ${
+    isDivorce ? 'marriage' : 'civil partnership'
+  }, which might affect the legal power (jurisdiction) of the court.`,
+  line19: `${
+    bothApplicantsLegalProceedings(userCase)
+      ? `Applicant 1 has given details of other court cases relating to the ${
+          isDivorce ? 'marriage' : 'civil partnership'
+        }:` +
+        '<br>' +
+        userCase?.applicant1LegalProceedingsDetails +
+        '<br><br>' +
+        `Applicant 2 has given details of other court cases relating to the ${
+          isDivorce ? 'marriage' : 'civil partnership'
+        }:` +
+        '<br>' +
+        userCase?.applicant2LegalProceedingsDetails
+      : !bothApplicantsLegalProceedings(userCase)
```

Comment:
!bothApplicantsLegalProceedings will return True if **either** applicant1LegalProceedings or applicant2LegalProceedings is No, so it won't work for the next line which is for if **both** are No